client/me/purchases/cancel-purchase/button.jsx

```diff
@@ -60,9 +60,38 @@ const CancelPurchaseButton = React.createClass( {
 					isPrimary: true,
 					disabled: this.state.submitting,
 					onClick: this.submitCancelAndRefundPurchase
-				},
+				}
 			];
 
+		let cancelationEffectString = this.translate(
+			'All plan features and custom changes will be removed from your site and you will be refunded %(priceText)s.', {
+				args: {
+					priceText: refundText
```

Comment:
This has come up before but again I don't think it's a good idea to use `priceText` in these strings because translators won't have a clear idea of what this variable will contain. So we should either add more context or (probably better) rename it into `cost` or something like that.